configure.ac

```diff
@@ -93,7 +93,7 @@ AC_PROG_CC
 AM_PROG_CC_C_O
 AC_PROG_CXX
 AC_PROG_CPP
-AC_PROG_LEX(yywrap)
+AC_PROG_LEX([yywrap])
```

Comment:
This probably could have been squashed into the original commit. No harm done though and probably not worth another round of CI builds just to do the squash.

Reply:
Resolved